frontend/public/components/pod-exec.jsx

```diff
@@ -144,10 +144,14 @@ export class PodExec extends React.PureComponent {
 
     return <div>
       <div className="co-m-pane__top-controls">
-        <span className="log-container-selector__text">
-          Connecting to
-        </span>
-        <Dropdown className="btn-group" items={_.mapValues(containers, nameWithIcon)} title={nameWithIcon(activeContainer || <LoadingInline />)} onChange={this.onChangeContainer} />
+        <div className="co-m-pane__top-controls__group--left">
```

Comment:
BEM suggests you shouldn't nest elements.  http://getbem.com/faq/#css-nested-elements